src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -67,9 +67,20 @@ public static partial class PlatformDetection
         public static bool IsS390xProcess => (int)RuntimeInformation.ProcessArchitecture == 5; // Architecture.S390x
         public static bool IsArmv6Process => (int)RuntimeInformation.ProcessArchitecture == 7; // Architecture.Armv6
         public static bool IsPpc64leProcess => (int)RuntimeInformation.ProcessArchitecture == 8; // Architecture.Ppc64le
```

Comment:
I have missed that most of these properties are defined using hardcoded constants. It would be nice to follow the same style for all for them - either convert them all or keep them all on the same plan.

Reply:
Ok, I'm not familiar with the support status of these hardcoded architectures. So I tend to remain consistent with other hardcoded architectures here to avoid the break of CI checks.
And these hardcoded constants can be modified together in right time in the future.